src/compiler/diagnosticMessages.json

```diff
@@ -1955,6 +1955,10 @@
         "category": "Error",
         "code": 2691
     },
+    "Cannot find name '{0}'. A type exists with this name, but no value.": {

```

Comment:
I like @DanielRosenwasser's recommendation here:
`'{0}' only refers to a type, but is being used as a value here.`